Source/WebCore/rendering/svg/RenderSVGResourceMasker.cpp

```diff
@@ -75,7 +67,25 @@ bool RenderSVGResourceMasker::applyResource(RenderElement& renderer, const Rende
     // Determine scale factor for the mask. The size of intermediate ImageBuffers shouldn't be bigger than kMaxFilterSize.
     ImageBuffer::sizeNeedsClamping(repaintRect.size(), scale);
 
-    if (!maskerData->maskImage && !repaintRect.isEmpty()) {
+    std::optional<FloatRect> objectBoundingBox;
+    if (maskElement().maskContentUnits() == SVGUnitTypes::SVG_UNIT_TYPE_OBJECTBOUNDINGBOX)
+        objectBoundingBox = renderer.objectBoundingBox();
```

Comment:
Instead of storing `std::optional<FloatRect>` for the `objectBoundingBox`, should not we store `std::optional<AffineTransform>` for `maskContentTransformation`? I see `inputs.objectBoundingBox` is only used when calculating `maskContentTransformation`.